assignments/prototype-refactor.js

```diff
@@ -7,3 +7,170 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+
+
+/*
+  === GameObject ===
+  * createdAt
+  * dimensions (These represent the character's size in the video game)
+  * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
+*/
+    class GameObject {
+        constructor(charAtt){
+            this.createdAt = charAtt.createdAt;
+            this.dimensions = charAtt.dimensions;
```

Comment:
What exactly is the dimensions attribute, and how would you get it's attributes individually?